dev/conductor/core/lib/src/state.dart

```diff
@@ -77,7 +77,7 @@ String presentState(pb.ConductorState state) {
   buffer.writeln('\tCurrent git HEAD: ${state.engine.currentGitHead}');
   buffer.writeln('\tPath to checkout: ${state.engine.checkoutPath}');
   buffer.writeln(
-      '\tPost-submit LUCI dashboard: ${luciConsoleLink(state.releaseChannel, 'engine')}');
+      '\tPost-submit cocoon dashboard: ${globals.cocoonDashboard}');
```

Comment:
We're in a weird state. For the framework, you need to look at both LUCI and the Cocoon dashboard.

I expect the next beta cut we will be able to deprecate the MILO dashboards for release CI. We'll still need it for packaging.

Reply:
This is the link for engine and framework post submits.  I typically utilize those dashboards for the post submit tests.  The packaging link has not changed and still utilizes the LUCI link. 

See next.dart line 340.

Reply:
To clarify, this change is good, as cocoon should be the source of truth for engine post-submit